scripts/benchmark_stats.py

```diff
@@ -8,8 +9,9 @@ class BenchmarkStats:
     PERCENTILES_TO_REPORT = (50, 90, 95, 99, 99.9)
     PERCENTILE_TO_USE_FOR_AVERAGES = 99
 
-    def __init__(self, folder_paths: List[str]):
+    def __init__(self, folder_paths: List[str], output_to_github: bool = False):
         self._files: List = []
+        self.output_to_github = output_to_github
```

Comment:
Minor, I think if we use here we won't need to pass this new var through the classes 
```suggestion
        self.output_to_github = os.getenv("OUTPUT_TO_GITHUB", False)
```

Reply:
Added now and also removed the Github var from the slack ci script. Thanks.